### PR TITLE
Fix queue error related to https://github.com/wildfly/wildfly/pull/15222

### PR DESCRIPTION
**Pull request by marcosgopen**
quickstart failure in CI: issue related to the queue (https://ci-jenkins-csb-narayana.apps.ocp-c1.prod.psi.redhat.com/job/narayana-quickstarts/jdk=openJDK11,label=jnlp-agent/80/console)
As discussed on zulip (https://wildfly.zulipchat.com/#narrow/stream/174184-wildfly-developers/topic/queue.20is.20missing.20.5Bjboss.2Enaming.2Econtext.2Ejava.2E.2E.2E.5D) the quickstart failure is related to a change in the Wildfly repo (https://github.com/wildfly/wildfly/pull/15222). To sum up the queue was not used properly in quickstarts and this had been allowed by Wildfly before this change. 
Using class 'org.jboss.as.arquillian.api.ServerSetupTask' that will setup the test Server with the activemq queue needed. 
Update of creaper, arquillian, wildfly-core and wildfly-loggin dependencies
-ds.xml file deployments are deprecated, so removing it.
depends on Narayana PR https://github.com/jbosstm/narayana/pull/2143

### CODE REVIEW

**Comment by mmusgrov** [comment]
@jmfinelli Will you be able to review this one, I ask because I think you did some work in this area when you were fixing JBTM-2524

**Comment by jmfinelli** [comment]
> @jmfinelli Will you be able to review this one, I ask because I think you did some work in this area when you were fixing JBTM-2524

Yep, no problem

File: pom.xml [hunk 1]
```diff
@@ -209,8 +209,8 @@
         <version.weld>5.0.1.Final</version.weld>
         
         <version.wildfly.plugin>4.0.0.Final</version.wildfly.plugin>
-        <version.wildfly.core>18.1.2.Final</version.wildfly.core>
-        <version.wildfly.logging>5.0.0.Beta5</version.wildfly.logging>
+        <version.wildfly.core>20.0.2.Final</version.wildfly.core>
+        <version.wildfly.logging>5.0.0.Final</version.wildfly.logging>
```
Comment (jmfinelli) [inline pom.xml:213]: Could you please move these version bumps to another commit? Thanks

File: jta-1_2-in-wildfly/src/test/java/org/jboss/narayana/quickstarts/jta/TestCase.java [hunk 2]
```diff
@@ -22,8 +30,35 @@
  * @author <a href="mailto:gytis@redhat.com">Gytis Trikleris</a>
  */
 @RunWith(Arquillian.class)
+@ServerSetup(value = TestCase.ServerCmrSetup.class)
 public class TestCase {
 
+    public static class ServerCmrSetup implements ServerSetupTask {
```
Comment (jmfinelli) [inline jta-1_2-in-wildfly/src/test/java/org/jboss/narayana/quickstarts/jta/TestCase.java:36]: Could you rename this class with a name that clearly explains its purpose?

File: rts/at/jta-service/jms/src/test/java/org/jboss/jbossts/resttxbridge/quickstart/jms/QueueResourceTest.java [hunk 3]
```diff
@@ -26,8 +33,32 @@
  * 
  */
 @RunWith(Arquillian.class)
+@ServerSetup(value = QueueResourceTest.ServerCmrSetup.class)
 public class QueueResourceTest {
 
+    public static class ServerCmrSetup implements ServerSetupTask {
```
Comment (jmfinelli) [inline rts/at/jta-service/jms/src/test/java/org/jboss/jbossts/resttxbridge/quickstart/jms/QueueResourceTest.java:39]: Could you rename this class with a name that clearly explains its purpose?

File: wildfly/commit-markable-resource/src/test/java/org/jboss/narayana/quickstarts/cmr/CmrLrcoTestCase.java [hunk 4]
```diff
@@ -50,6 +50,8 @@ public void setup(ManagementClient managementClient, String containerId) throws
 
             try(FailuresAllowedBlock allowedBlock = creaper.allowFailures()) {
                 creaper.execute("/subsystem=transactions/commit-markable-resource=\"java:jboss/datasources/jdbc-cmr\":add()");
+                creaper.execute(
+                        "/subsystem=messaging-activemq/server=default/jms-queue=\"cmr\":add(entries=[java:/queue/cmr])");
```
Comment (jmfinelli) [inline wildfly/commit-markable-resource/src/test/java/org/jboss/narayana/quickstarts/cmr/CmrLrcoTestCase.java:54]: Is the new line needed? 

You can ignore this comment... :see_no_evil:

**Review by jmfinelli — APPROVED** [review]
Although I added a couple of comments, LGTM!

File: wildfly/commit-markable-resource/src/test/java/org/jboss/narayana/quickstarts/cmr/CmrLrcoTestCase.java (hunk 4 above)
Comment (mmusgrov) [reply wildfly/commit-markable-resource/src/test/java/org/jboss/narayana/quickstarts/cmr/CmrLrcoTestCase.java:54]: Agreed: we have line length limitations which necessitates adding them, although I forget what it is - when I started out it used to be 80 characters to accommodate the old VT 100 type computer terminals but I think the recommended length is a bit longer now - do we have anything in our checkstylesheet?

Comment (marcosgopen) [reply wildfly/commit-markable-resource/src/test/java/org/jboss/narayana/quickstarts/cmr/CmrLrcoTestCase.java:54]: I set an automatic new line when the line is longer than 120 chars (IIRC) to be more readable.

File: jta-1_2-in-wildfly/src/test/java/org/jboss/narayana/quickstarts/jta/TestCase.java (hunk 2 above)
Comment (marcosgopen) [reply jta-1_2-in-wildfly/src/test/java/org/jboss/narayana/quickstarts/jta/TestCase.java:36]: Thanks @jmfinelli , my mistake. Fixing it.

File: pom.xml (hunk 1 above)
Comment (marcosgopen) [reply pom.xml:213]: Thanks @jmfinelli , sure!

File: wildfly/commit-markable-resource/src/test/java/org/jboss/narayana/quickstarts/cmr/CmrLrcoTestCase.java (hunk 4 above)
Comment (marcosgopen) [reply wildfly/commit-markable-resource/src/test/java/org/jboss/narayana/quickstarts/cmr/CmrLrcoTestCase.java:54]: @mmusgrov I cannot see anything related to length limitations here (https://github.com/wildfly/wildfly-checkstyle-config/blob/master/src/main/resources/wildfly-checkstyle/checkstyle.xml). Maybe somewhere else?
